guides/07_streaming/02_object-detection-from-webcam.md

```diff
@@ -1,93 +1,107 @@
-# Object Detection from a Webcam Stream
+# Real Time Object Detection from a Webcam Stream with WebRTC
 
 Tags: VISION, STREAMING, WEBCAM
 
-In this guide we'll use Yolo-v10 to do near-real time object detection in Gradio from a user's webcam.
-Along the way, we'll be using the latest streaming features introduced in Gradio 5.0.
+In this guide, we'll use YOLOv10 to perform real-time object detection in Gradio from a user's webcam feed. We'll utilize the latest streaming features introduced in Gradio 5.0. You can see the finished product in action below:
 
-## Setting up the Model
+![WebRTC Object Detection Demo](https://github.com/user-attachments/assets/4584cec6-8c1a-401b-9b61-a4fe0718b558)
```

Comment:
Video seems to be broken:
<img width="906" alt="image" src="https://github.com/user-
attachments/assets/03647966-91fd-4b53-8b6c-40ca8798451b">

Reply:
Thank you - had to embed as a <video> directly